plugin.video.tv3.cat/changelog.txt

```diff
@@ -1,4 +1,9 @@
-﻿v1.4.0
+﻿v1.4.1
+- Corregits canvis al website. Corregits errors en les seccions "Avui destaquem" i "Col·leccions"
```

Comment:
Changelog in gotham is presented to the user. Please keep it in english only.

Reply:
Ok, I will fix it today